source/Music.h

```diff
@@ -47,6 +47,9 @@ class Music {
 	// Get the next audio buffer to play.
 	const std::vector<int16_t> &NextChunk();
 
+	bool IsFinished() const;
+	void Finish();
```

Comment:
we using this one?
should we be using it?

Reply:
It does not appear to be used but it makes sense to have a setter where you have a getter to short-cut it.  This engine support is more "initial support for music" by RisingLeaf and I polished it a little bit fixing bugs where I found them.  I think it should probably stay available as we expand on music support with new features.

For example, event-based music starting should stop existing music if it has a higher priority.